source/macos/AdaptiveCards/AdaptiveCards/Renderers/TextInputRenderer.swift

```diff
@@ -74,7 +71,7 @@ class TextInputRenderer: NSObject, BaseCardElementRendererProtocol {
             textView.cell?.truncatesLastVisibleLine = true
             textView.cell?.lineBreakMode = .byTruncatingTail
             textView.isHidden = !inputBlock.getIsVisible()
-            textView.setAccessibilityTitle(config.localisedStringConfig.inputTextFieldAccessibilityTitle)
+            textView.setAccessibilityRoleDescription(config.localisedStringConfig.inputTextFieldAccessibilityTitle)
         }
```

Comment:
So, before `accessibilityTitle` was working and now its working?
Maybe your change (due to overrides) is causing it to break?

Reply:
Yes, plus, the titles we're supplying are actually RoleDescriptors only, so it makes sense to assign it as roleDescriptor itself, right?